packages/suite/src/components/suite/Banners/SafetyChecks.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Translation } from '@suite-components';
+import { useActions } from '@suite-hooks';
+import Wrapper from './components/Wrapper';
+import * as routerActions from '@suite/actions/suite/routerActions';
+
+interface Props {
+    onDismiss?: () => void;
+}
+
+const SafetyChecksBanner = (props: Props) => {
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    return (
+        <Wrapper
+            variant="warning"
+            body={<Translation id="TR_SAFETY_CHECKS_DISABLED_WARNING" />}
+            action={{
+                label: <Translation id="TR_SAFETY_CHECKS_BANNER_CHANGE" />,
+                // TODO: Use anchor to bring the user to the appropriate section of settings.
+                onClick: () => goto('settings-device'),
+                'data-test': '@banner/safety-checks/button',
+            }}
+            dismissal={
+                props.onDismiss !== undefined
+                    ? {
+                          onClick: props.onDismiss,
+                          'data-test': '@banner/safety-checks/dismiss',
+                      }
+                    : undefined
```

Comment:
```suggestion
                props.onDismiss !== undefined && {
                          onClick: props.onDismiss,
                          'data-test': '@banner/safety-checks/dismiss',
                      }
```
I think this is the same and shorter. (needs formatting)

Reply:
This doesn't type check - the type of that expression is `{...} | false` but `{...} | unefined` is required. The ternary can still be rewritten as
```suggestion
                props.onDismiss !== undefined && {
                          onClick: props.onDismiss,
                          'data-test': '@banner/safety-checks/dismiss',
                } || undefined
                 
```

modulo formatting. LMK if you like that better.

Reply:
Assuming your 👍 means 'yes' here's  a [fixup](https://github.com/trezor/trezor-suite/pull/3681/commits/6607e363d2aad64456f156adcfe576add25302c9).